### ISSUE TITLE
Branch names are not displayed

### ISSUE THREAD

**Issue by jacksleight** [simonhamp/Gitamic#36]
**Describe the bug**

The branch names are not displaying correctly in the Gitamic interface for some reason. Everything seems to work absolutely fine otherwise.

**To Reproduce**

1. Go to the Gitamic page in the CP

**Expected Behaviour**

I think the current branch name is supposed to appear.

**Screenshots**

![Screenshot 2022-05-11 at 11 23 41](https://user-images.githubusercontent.com/126740/167828488-ceee9558-1bd8-40df-93ab-4d4f634b3588.png)

**Versions:**
 - PHP: 8.1.2
 - Browser: Chrome 101
 - Statamic: 3.3.10
 - Gitamic release: 1.8.4
 - Gitamic version: basic

**Comment by jacksleight** [comment]
Actually I've just noticed the Push button also isn't visible, even though I have a commit pending. Could this be some kind of issue with connecting to the remote repo?

**Comment by jacksleight** [comment]
This was due to a permissions issue with my SSH key, which caused the `git fetch` command to fail.

In case anyone else runs into the same problem I did: 

PHP was set up to run under the correct user, and that user had the correct ssh keys installed. The problem was that the key was protected with a passphrase, and that doesn't work through PHP. Maybe it's possible to set that up (I have no idea), but I just removed the passphrase from the key and that sorted it.
